src/form-item.scss

```diff
@@ -11,25 +11,25 @@ $block: #{$fd-namespace}-form-item;
 .#{$block} {
   @include fd-reset();
 
-  margin-bottom: fd-space("small");
+  display: flex;
+  flex-direction: column;
 
-  &--inline {
-    float: left;
-    margin-right: fd-space(4);
+  &--horizontal {
```

Comment:
What are the different use cases for `--horizontal` versus `--inline`?

Reply:
`--inline` is what we had before for displaying the checkboxes and radio buttons inline.
<img width="376" alt="Screen Shot 2019-11-13 at 2 30 45 PM" src="https://user-images.githubusercontent.com/39598672/68797313-60265100-0622-11ea-87e9-95818b67ec0a.png">
This was only the elements go to one line, the label is on another line. 

But then we needed to add inline items - label and input inline. This was achieved by using flexbox. 
<img width="665" alt="Screen Shot 2019-11-13 at 2 31 06 PM" src="https://user-images.githubusercontent.com/39598672/68797462-a8de0a00-0622-11ea-80a9-6300d362fbe1.png">

Stefano is working currently on Forms Layout - there's going to be vertical and horizontal layout. 

So, to minimize the breaking changes, we decided to keep the `--inline` modifier for displaying the checkboxes and radio btns inline, and add `--horizontal` for displaying the items inline